src/index.js

```diff
@@ -50,6 +56,17 @@ const MODES = {
     // when upgrading
     manageAppLifecycle: false,
   },
+  satellite: {
+    name: 'satellite',
+    deployments: '/srv/software',
+    appsToDeploy: ['medic-api'],
+    upgradeDocuments: [`_design/medic`],
```

Comment:
Note that this design document will change in situations that are not a deployment, because settings are written into this design document. 

Long term they shouldn't be, but that's the situation we live in right now.

So following could be OK, but you'd only want to fire satellite restarts if you detected you should, which is tricky.

We could also consider following the deploy document, which would probably be less flaky, as changes to that will indicate deployments, and you can even tell from the log what step it's up to, and you could perform different actions at different stages. This is a complicated phone discussion, as is my other large comment on this PR :-)

Reply:
I believe horti already only fires restarts if the app changes:
https://github.com/medic/horticulturalist/pull/26/files/15f23a71b68e91a08c95787ecef2bdf0a529b024#diff-10148fb1a5d164883ca8a2b1f9196294R130